templates/cloud_welcome_email.html

```diff
@@ -1,169 +1,173 @@
+                                            {{.Props.GettingStartedQuestions}} <a
+                                                style="text-decoration: none; color: #6DA4EB;"
+                                                href="mailto:support@mattermost.com">support@mattermost.com.</a>
                                         </p>
-                                    </div>
-                                </td>
-                            </tr>
-                        </table>
-                        <table border="0" cellpadding="0" cellspacing="0" style="margin-top: 50px;">
-                            <tr>
-                                <td>
-                                    <p
-                                        style="font-family: Open Sans; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40;">
-                                        {{.Props.GettingStartedQuestions}} <a
-                                            style="text-decoration: none; color: #6DA4EB;"
-                                            href="mailto:support@mattermost.com">support@mattermost.com.</a>
-                                    </p>
 
-                                </td>
-                            </tr>
-                        </table>
-                        <table align="center" border="0" cellpadding="0" cellspacing="0">
-                            <tr>
-                                <td style="padding: 20px 20px 10px; text-align:center;">
-                                    <img src="{{.Props.WorkSpacePath}}/static/images/invite_illustration.png"
-                                        width="320px" alt="invite_illustration">
+                                    </td>
+                                </tr>
+                            </table>
+                            <table align="center" border="0" cellpadding="0" cellspacing="0">
+                                <tr>
+                                    <td style="padding: 20px 20px 10px; text-align:center;">
+                                        <img src="{{.Props.WorkSpacePath}}/static/images/invite_illustration.png"
+                                            width="320px" alt="invite_illustration">
+
+                                    </td>
+                                </tr>
+                            </table>
+                            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td>
+                                        <p
+                                            style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 12px; line-height: 16px; text-align: center; color: rgba(61, 60, 64, 0.56);">
+                                            © 2021 Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA,
```

Comment:
There's a translation for this. Doesn't really translate to different languages but let's us change the year in one place and have it propagate to all emails at once.

Reply:
In fact, I think there's an entire email footer template you might be able to just insert here.